evmcore/dummy_block.go

```diff
@@ -138,14 +149,13 @@ type EvmHeaderJson struct {
 	TxHash      common.Hash      `json:"transactionsRoot" gencodec:"required"`
 	ReceiptHash common.Hash      `json:"receiptsRoot"     gencodec:"required"`
 	Bloom       types.Bloom      `json:"logsBloom"        gencodec:"required"`
-	Difficulty  *hexutil.Big     `json:"difficulty"       gencodec:"required"`
 	Number      *hexutil.Big     `json:"number"           gencodec:"required"`
 	GasLimit    hexutil.Uint64   `json:"gasLimit"         gencodec:"required"`
 	GasUsed     hexutil.Uint64   `json:"gasUsed"          gencodec:"required"`
 	Time        hexutil.Uint64   `json:"timestamp"        gencodec:"required"`
 	TimeNano    hexutil.Uint64   `json:"timestampNano"`
 	Extra       hexutil.Bytes    `json:"extraData"        gencodec:"required"`
-	MixDigest   common.Hash      `json:"mixHash"`
+	PrevRandao  common.Hash      `json:"prevRandao"`
```

Comment:
If I read [EIP-4399](https://eips.ethereum.org/EIPS/eip-4399#block-structure) correctly, in the JSON file format the field is still to be named `mixHash`. Just internally, the renaming is suggested.

This Json format is the format we use for responding to RPC queries. In [docs](https://www.quicknode.com/docs/kaia/eth_getHeaderByNumber) this field is called `mixHash`.